src/main/java/com/metsci/laproc/ROCCurvePlot.java

```diff
@@ -32,7 +32,7 @@ public SimplePlot2D getLayout()
         Axis yAxis = graph.getYAxis();
 
         // set axis labels and chart title
-        plot.setTitle( "ROC Curve Example" );
```

Comment:
This should probably say Curve not SimpleGraphableData (silly intellij refactoring)